### PR TITLE
[admin] Pull latest from DefinitelyTyped

### PR DESCRIPTION
**Pull request by alison-mk**

### CODE REVIEW

File: generate-docs/json/excel_1_11/excel.api.json [hunk 1]
```diff
@@ -225456,6 +225467,21 @@
                 }
               ]
             },
+            {
+              "kind": "Class",
+              "canonicalReference": "excel!Excel.RangeCustom:class",
```
Comment (AlexJerabek) [inline generate-docs/json/excel_1_11/excel.api.json:225472]: I think the RangeCustom class slipped into DefinitelyTyped. Let's remove that.

File: generate-docs/json/excel_1_11/excel.api.json [hunk 2]
```diff
@@ -222267,6 +222236,48 @@
               },
               "implementsTokenRanges": []
             },
+            {
+              "kind": "Interface",
+              "canonicalReference": "excel!Excel.Range:interface",
```
Comment (AlexJerabek) [inline generate-docs/json/excel_1_11/excel.api.json:222241]: I think this empty Range interface is also tied to the RangeCustom class.

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Looks like that rogue RangeCustom class was accidentally added to the d.ts. Can we get rid of that at the DefinitelyTyped level and regenerate the ref docs?

File: generate-docs/json/excel_1_11/excel.api.json (hunk 1 above)
Comment (alison-mk) [reply generate-docs/json/excel_1_11/excel.api.json:225472]: Thanks for catching this! Working on the fix now.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**